app/Http/Controllers/ApiController.php

```diff
@@ -18,13 +18,13 @@ public function __construct()
     public function getStudentDetails(Request $request)
     {
         if (!($request->has("username"))) {
-            return json_encode(array("status" => "failed",
-                "message" => "missing username"));
+            $userResponse = array("status" => "Fail", "message" => "missing username");
+            return response()-> json($userResponse, 400);
         }
 
         if (!($request->has("password"))) {
-            return json_encode(array("status" => "failed",
-                "message" => "missing password"));
+            $passResponse = array("status" => "Fail", "message" => "missing password");
```

Comment:
Same here. Let's just call it `$response`.
Also, `fail` should be in lowercase.